gqlalchemy/connection.py

```diff
@@ -107,21 +107,37 @@ def _create_connection(self) -> Connection:
 def _convert_memgraph_value(value: Any) -> Any:
     """Converts Memgraph objects to custom Node/Relationship objects"""
     if isinstance(value, mgclient.Relationship):
-        return Relationship(
-            rel_id=value.id,
-            rel_type=value.type,
-            start_node=value.start_id,
-            end_node=value.end_id,
-            properties=value.properties,
+        return Relationship.parse_obj(
+            {
+                "type": value.type,
+                "_id": value.id,
+                "_relationship_id": value.id,
+                "_relationship_type": value.type,
+                "_start_node_id": value.start_id,
+                "_end_node_id": value.end_id,
+                "_properties": value.properties,
+                **value.properties,
```

Comment:
You're seeing correctly that things are duplicated, this is because discussions in the repository haven't been resolved, which ever approach we take, we can easily update.

The `_properties` field in particular has to be removed regardless of these discussions, because it won't be updated when the user changes an individual property.

Reply:
`_properties` has been converted to a view.